pkgs/stdenv/generic/setup.sh

```diff
@@ -1527,6 +1517,7 @@ showPhaseFooter() {
     local startTime="$2"
     local endTime="$3"
     local delta=$(( endTime - startTime ))
+    echo "finished phase: $phase"
```

Comment:
```suggestion
    echo -n "finished phase: $phase in "
```

Reply:
@Artturin , thank you for review. I am not sure if I understand your suggestion. Do you want to always print `$phase completed in` ? If that's true, then we need to remove line saying `(( delta < 30 )) && return`.

Reply:
either we print in it always on the same line or on the same line when the time is above 30 seconds like now

Reply:
Thanks,

I changed `showPhaseFooter` output to always duration of the phase. 

Now we have output like this:
```
nix-build -A hello 2>&1 | grep "finished phase"

finished phase: unpackPhase in 0 seconds
finished phase: patchPhase in 0 seconds
finished phase: updateAutotoolsGnuConfigScriptsPhase in 0 seconds
finished phase: configurePhase in 13 seconds
finished phase: configurePhase in 1 minutes 2 seconds
finished phase: buildPhase in 15 seconds
```

It looks to me that always having duration info can be be helpful for debugging. I hope it doesn't slow things down.